infra/eligibility_screener/template/ecs.tf

```diff
@@ -28,6 +40,89 @@ resource "aws_security_group" "allow-screener-traffic" {
   }
 }
 
+resource "aws_security_group" "allow-lb-traffic" {
+  name        = "screener_load_balancer_sg"
+  description = "Allows load balancers to communicate with tasks"
+  vpc_id      = module.constants.vpc_id
+
+  ingress {
+    description      = "HTTP traffic from VPC"
+    from_port        = 80
+    to_port          = 80
+    protocol         = "tcp"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+
+  ingress {
+    description = "allow lb traffic"
+    from_port   = 0
+    to_port     = 65535
+    protocol    = "tcp"
+    security_groups = [
+      "sg-0c50cf775611d9db2",
+    ]
+  }
+  egress {
+    description      = "allow all outbound traffic from screener"
+    from_port        = 0
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+}
+
+# ---------------------------------------
+#
+# Load Balancing
+#
+# ---------------------------------------
+
+resource "aws_lb" "eligibility-screener" {
+  name               = "${var.environment_name}-screener-lb"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = [aws_security_group.allow-lb-traffic.id]
+  subnets = [
+    "subnet-05b0618f4ef1a808c",
+    "subnet-06067596a1f981034",
+    "subnet-06b4ec8ff6311f69d",
+    "subnet-08d7f1f9802fd20c4",
+    "subnet-09c317466f27bb9bb",
+    "subnet-0ccc97c07aa49a0ae"
+  ] # find a way to map all the default ones here; hardcoding for now
+  ip_address_type        = "ipv4"
+  desync_mitigation_mode = "defensive"
```

Comment:
didn't know about this, pretty cool 👍

Reply:
[More reading for other folks](https://aws.amazon.com/about-aws/whats-new/2020/08/application-and-classic-load-balancers-adding-defense-in-depth-with-introduction-of-desync-mitigation-mode/#:~:text=In%20the%20%E2%80%9CDefensive%E2%80%9D%20mode%2C%20the%20load%20balancer%20will%20perform%20three%20specific%20tasks.)